packages/@spectrum-icons/illustrations/src/Upload.tsx

```diff
@@ -1,12 +1,14 @@
-import {getIllustrationProps, IllustrationProps} from './utils';
```

Comment:
is `getIllustrationProps` unused now? should it be deleted?

Reply:
I think so, I'll delete that since it isn't public